src/agent/browser_use/browser_use_agent.py

```
@@ -97,22 +98,58 @@ async def run(
-                    step_delay_minutes = float(step_delay_minutes_str)
-                    if step_delay_minutes > 0.0:
-                        step_delay_seconds = step_delay_minutes * 60
-                        logger.info(
-                            f"Waiting for {step_delay_seconds} seconds ({step_delay_minutes} minutes) before next step..."
+                # Step delay logic
+                enable_random_step_delay_str = os.environ.get("STEP_ENABLE_RANDOM_INTERVAL", "false")
+                enable_random_step_delay = enable_random_step_delay_str.lower() == "true"
+
+                if enable_random_step_delay:
+                    min_delay_minutes_str = os.environ.get("STEP_MIN_DELAY_MINUTES", "0.0")
+                    if not min_delay_minutes_str: min_delay_minutes_str = "0.0"
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)